### ISSUE TITLE
Bug :Jest snapshot testing fails because fo random aria-controls each time in '@progress/kendo-react-dateinputs';

### ISSUE THREAD

**Issue by SriPalakodety** [telerik/kendo-react#502]
TimePicker component from '@progress/kendo-react-dateinputs' uses random generated aria-controls every time (dependent on  "@progress/kendo-react-common") and as a result of this, jest snapshot testing fails. Please address this issue

**Comment by SriPalakodety** [comment]
Found another way to test it(using enzyme's mount)
